net/curl/Portfile

```diff
@@ -209,6 +209,15 @@ if {${name} eq ${subport}} {
         default_variants-append +http2
     }
 
+    variant http3 requires gnutls description {Support HTTP/3 with nghttp3 and ngtcp2} {
+        depends_lib-append      port:nghttp3 \
+                                port:ngtcp2
+        configure.args-append   --with-nghttp3=${prefix} \
+                                --with-ngtcp2=${prefix}
+
+        patchfiles-append       patch-lib-vquic-curl_ngtcp2.c.diff
```

Comment:
It looks like you forgot to add the patchfile?

Reply:
Oops! Sure did.